pkg/expr/sql/parser_allow.go

```diff
@@ -210,6 +210,12 @@ func allowedFunction(f *sqlparser.FuncExpr) (b bool) {
 	case "cast":
 		return
 
+	// JSON functions
+	case "json_extract", "json_unquote", "json_contains",
+		"json_object", "json_array", "json_set", "json_remove",
+		"json_length", "json_search", "json_type":
```

Comment:
I'd like to see us TDD adding these, personally. I'd like to see a test case in `parser_allow_test`, and for that test-case to be runnable in SQL-Fiddle, without and dependency on any pre-existing table or column.